perfrunbook/utilities/install_perfrunbook_dependencies_al2.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
```

Comment:
unify into one script that detects redhat-like or debian-like and does the right thing